js/test/unit/builders/utils.ts

```diff
@@ -32,22 +32,14 @@ const randnulls = <T, TNull = null>(values: T[], n: TNull = <any>null) => values
 export const randomBytes = (length: number) => fillRandom(Uint8Array, length);
 
 export const stringsNoNulls = (length = 20) => Array.from({ length }, (_) => randomString(1 + (Math.trunc(Math.random() * 19))));
-export const timestamp32sNoNulls = (length = 20, now = Math.trunc(Date.now() / 86400000)) =>
+export const timestampNoNulls = (length = 20, now = Math.trunc(Date.now() / 86400000)) =>
     Array.from({ length }, (_) => (Math.trunc(now + (rand() * 10000 * (rand() > 0.5 ? -1 : 1)))) * 86400000);
```

Comment:
Given the scale of `rand() * 10000` compared to `now`, this will never produce timestamps with a negative value, right?

Reply:
Good catch. I changed it to use `100000` which gives us negative numbers as well.